src/wasm/kotlin/java/arcs/entity-class.kt

```diff
@@ -0,0 +1,179 @@
+// This is a reference for the entity classes generated by schema2wasm, based on the schema:
+//
+// particle Test
+//   data: reads {num: Number, txt: Text, lnk: URL, flg: Boolean, ref: &{val: Text}}
+
+@file:Suppress("PackageName", "TopLevelName")
+package arcs
+
+//
+// GENERATED CODE -- DO NOT EDIT
+//
+// Current implementation doesn't support references or optional field detection
+
+class Test_Data_Ref() : Entity<Test_Data_Ref>() {
```

Comment:
```
class Test_Data_Ref()  // Implementation omitted for brevity
```

The "main" class Test_Data showcases the entity API for each field type. Having the Ref class impl just makes the file more noisy.